ctyparser/bigcty.py

```diff
@@ -92,32 +114,56 @@ def import_dat(self, dat_file: Union[str, os.PathLike]) -> None:
         with dat_file.open("r") as file:
             cty_dict = dict()
 
+            # get the version from the file, set filepointer to beginning of file afterwards
```

Comment:
This is fairly self-explanatory. Comment not needed.

```suggestion
            # get the version from the file
```